common/src/main/java/com/wynntils/handlers/actionbar/ActionBarHandler.java

```diff
@@ -102,6 +102,8 @@ private void processPosition(ActionBarPosition pos, Map<ActionBarPosition, Style
                 } else {
                     segment.update(m);
                 }
+
+                break;
```

Comment:
Is this a sane change @magicus? I believe it should be correct since a position can only be 1 type of segment?

Reply:
(This ended up being unrelated to the fix, but it caught my attention..)

Reply:
Yes, it looks good. Since we have 3 segments, it is really a micro-optimization :-) but sure, I think it is correct.